### DISCUSSION TITLE
Use local assets when available

### DISCUSSION THREAD

**Discussion by pafgoncalves** [antonym/netboot.xyz#1100] [Q&A]
When i pull selected assets to the local assets i would expect that the corresponding menus changed to the local files.
Instead i have to change the menus manually.

Am i missing something or is this the expected behavior?

**Comment by antonym** [comment]
endpoints.yml dictates what assets can be pulled down.  When you switch menus to a new release or latest release, it'll pull down the latest menu release zip with the latest endpoints.yml that references the local assets.
